src/tiled/command.cpp

```diff
@@ -216,8 +215,14 @@ CommandProcess::CommandProcess(const Command &command, bool inTerminal, bool sho
 
     connect(this, SIGNAL(finished(int)), SLOT(deleteLater()));
 
-    if (showOutput)
+    if (showOutput) {
+        CommandManager::instance()->logger()->log(LoggingInterface::INFO,
+                                                  QString(QLatin1String("Executing: "))
```

Comment:
This should really be a translatable string, like `tr("Executing: %1").arg(...)`